Source/arcadia/core.clj

```diff
@@ -1,5 +1,7 @@
 (ns arcadia.core
   (:require [clojure.string :as string]
+            [clojure.core.server]
+            [arcadia.internal.state-help]
```

Comment:
yeah, could be moved to `all-user-namespaces` but thought here was a good place to remember them if those deps ever go away